common/log/sys_logger.go

```diff
@@ -120,7 +120,7 @@ func getSyslogLevel(lvl common.LogLevel) syslog.Priority {
 func (l *SysLogger) write(lvl string, format string, args ...interface{}) {
 	_, fn, ln, _ := runtime.Caller(3)
 	msg := fmt.Sprintf(format, args...)
-	l.logger.Print(lvl, " [", filepath.Base(fn), " (", ln, ")]: ", msg)
+	l.logger.Print("[", common.MountPath, "] ", lvl, " [", filepath.Base(fn), " (", ln, ")]: ", msg)
```

Comment:
Instead of dumping path after the pid and all, can we do something like [pid:path] and then rest of the log remains the same. This help us identify that pid is associated with the mount path.

Reply:
Syslog computes the time and pid internally I think because I couldn't find any code written by us for it. I can make this change in base log, but I just tried to keep the format consistent.